### PR TITLE
Conclude oSEAT3

### PR DESCRIPTION
**Pull request by fajar13k**
End of osu! South East Asia Tournament 3

### CODE REVIEW

File: wiki/Tournaments/oSEAT/2/en.md [hunk 1]
```diff
@@ -1,46 +1,43 @@
...
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter tag, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu!supporter tag |
 | *4th place* | 1 month of osu!supporter tag |
 | *5th place* | 1 month of osu!supporter tag |
 
 *Note: 5th place was determined by a playoff.*
 
-![oSEAT 2 Winner Badge](img/badge.png)
+![oSEAT2 winner badge](img/badge.png "SOFT4 Winner Badge")
```
Comment (MegaApplePi) [inline wiki/Tournaments/oSEAT/2/en.md]:
```suggestion
![oSEAT2 winner badge](img/badge.png "oSEAT2 Winner Badge")
```
typo?

File: wiki/Tournaments/oSEAT/3/en.md [hunk 2]
```diff
@@ -32,16 +32,16 @@ The **osu! South East Asia Tournament 3** (***oSEAT3***) is a South East Asian d
...
-| 4th place | 2 months of osu!supporter tag |
-| 5th place | 1 month of osu!supporter tag |
+| *4th place* | 2 months of osu!supporter tag |
+| *5th place* | 1 month of osu!supporter tag |
 
-*Note: 5th place will be determined by a playoff.*
+*Note: 5th place was determined by a playoff.*
 
-![oSEAT3 winner badge](img/badge.png)
+![oSEAT3 winner badge](img/badge.png "SOFT4 Winner Badge")
```
Comment (MegaApplePi) [inline wiki/Tournaments/oSEAT/3/en.md]:
```suggestion
![oSEAT3 winner badge](img/badge.png "oSEAT3 Winner Badge")
```

File: wiki/Tournaments/oSEAT/2/en.md [hunk 3]
```diff
@@ -1,46 +1,43 @@
...
+  - oSEAT2
 ---
 
 # osu! South East Asia Tournament 2
 
-![oSEAT 2 Logo](img/logo.png)
+![oSEAT2 logo](img/logo.png)
 
-The **osu! South East Asia Tournament 2** (***oSEAT 2***) was a South East Asian players' double-elimination one-on-one osu! tournament hosted by ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), ![][flag_ID] [nya10](https://osu.ppy.sh/users/1372608), and ![][flag_TH] [Trigonoculus](https://osu.ppy.sh/users/7627013). It was the second installment of the osu! South East Asia Tournament.
+The **osu! South East Asia Tournament 2** (***oSEAT2***) was a South East Asian players' double-elimination one-on-one osu! tournament hosted by ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), ![][flag_ID] [nya10](https://osu.ppy.sh/users/1372608), and ![][flag_TH] [Trigonoculus](https://osu.ppy.sh/users/7627013). This was the second installment of the osu! South East Asia Tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/2/en.md]:
```suggestion
The **osu! South East Asia Tournament 2** (***oSEAT2***) was a South East Asian players' double-elimination one-on-one osu! tournament hosted by ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), ![][flag_ID] [nya10](https://osu.ppy.sh/users/1372608), and ![][flag_TH] [Trigonoculus](https://osu.ppy.sh/users/7627013). It was the second installment of the osu! South East Asia Tournament.
```

File: wiki/Tournaments/oSEAT/2/en.md [hunk 4]
```diff
@@ -1,46 +1,43 @@
...
-| Grand Finals | 2019-01-26/2019-01-27 |
+| Semifinals | 2019-01-11/2019-01-13 |
+| Finals | 2019-01-18/2019-01-20 |
+| Grand Finals | 2019-01-27 |
 
 ## Prizes
 
-In every tournament, people conquering a place on the podium are eligible to receive unique prizes created specifically for osu! South East Asia Tournament 2 winners. It is possible that these items change with every installment of the osu! South East Asia Tournament.
-
 | Placing | Prize(s) |
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/2/en.md]:
```suggestion
| Placing | Prizes |
```

File: wiki/Tournaments/oSEAT/3/en.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ tags:
 
 ![oSEAT3 logo](img/logo.jpg)
 
-The **osu! South East Asia Tournament 3** (***oSEAT3***) is a South East Asian double-elimination 1v1 osu! tournament hosted by ![][flag_MY] [Kumaxione](https://osu.ppy.sh/users/2582531), ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), and ![][flag_VN] [steve\_04\_](https://osu.ppy.sh/users/10852911). This is the third installment of the osu! South East Asia Tournament.
+The **osu! South East Asia Tournament 3** (***oSEAT3***) was a South East Asian double-elimination 1v1 osu! tournament hosted by ![][flag_MY] [Kumaxione](https://osu.ppy.sh/users/2582531), ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), and ![][flag_VN] [steve\_04\_](https://osu.ppy.sh/users/10852911). This was the third installment of the osu! South East Asia Tournament.
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/3/en.md]:
```suggestion
The **osu! South East Asia Tournament 3** (***oSEAT3***) was a South East Asian double-elimination one-on-one osu! tournament hosted by ![][flag_MY] [Kumaxione](https://osu.ppy.sh/users/2582531), ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), and ![][flag_VN] [steve\_04\_](https://osu.ppy.sh/users/10852911). It was the third installment of the osu! South East Asia Tournament.
```

File: wiki/Tournaments/oSEAT/2/en.md [hunk 6]
```diff
@@ -1,46 +1,43 @@
...
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter tag, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter tag |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu!supporter tag |
 | *4th place* | 1 month of osu!supporter tag |
 | *5th place* | 1 month of osu!supporter tag |
 
 *Note: 5th place was determined by a playoff.*
 
-![oSEAT 2 Winner Badge](img/badge.png)
+![oSEAT2 winner badge](img/badge.png "oSEAT2 Winner Badge")
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/2/en.md]:
```suggestion
![oSEAT2 winner badge](img/badge.png)
```

File: wiki/Tournaments/oSEAT/3/en.md [hunk 7]
```diff
@@ -32,16 +32,16 @@ The **osu! South East Asia Tournament 3** (***oSEAT3***) is a South East Asian d
...
-| 4th place | 2 months of osu!supporter tag |
-| 5th place | 1 month of osu!supporter tag |
+| *4th place* | 2 months of osu!supporter tag |
+| *5th place* | 1 month of osu!supporter tag |
 
-*Note: 5th place will be determined by a playoff.*
+*Note: 5th place was determined by a playoff.*
 
-![oSEAT3 winner badge](img/badge.png)
+![oSEAT3 winner badge](img/badge.png "oSEAT3 Winner Badge")
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/3/en.md]:
```suggestion
![oSEAT3 winner badge](img/badge.png)
```

File: wiki/Tournaments/oSEAT/3/en.md [hunk 8]
```diff
@@ -94,8 +96,53 @@ The osu! South East Asia 3 is run by various community members.
 | O | ![][flag_SG] [moosepi](https://osu.ppy.sh/users/1868745) | ![][flag_ID] [Ascaveth](https://osu.ppy.sh/users/3245206) | ![][flag_MY] [Chiyuu](https://osu.ppy.sh/users/8226107) | ![][flag_ID] [thefarrel007](https://osu.ppy.sh/users/2429128) | ![][flag_TH] [muramax](https://osu.ppy.sh/users/10552306) | ![][flag_ID] [D-Rain](https://osu.ppy.sh/users/9925149) |
 | P | ![][flag_SG] [GSBlank](https://osu.ppy.sh/users/2312106) | ![][flag_SG] [woahsia](https://osu.ppy.sh/users/195946) | ![][flag_MY] [blufir\_](https://osu.ppy.sh/users/11134301) | ![][flag_MY] [Tokama Club](https://osu.ppy.sh/users/7321060) | ![][flag_SG] [Hecatia](https://osu.ppy.sh/users/8244635) | ![][flag_MY] [ManzMalaya](https://osu.ppy.sh/users/3999924) |
 
+---
+
+## Podium
+
+ This competition has come to an end and resulted in the following podium:
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/3/en.md]:
```suggestion
This competition has come to an end and resulted in the following podium:
```

File: wiki/Tournaments/oSEAT/2/en.md [hunk 9]
```diff
@@ -1,46 +1,43 @@
...
 
 ## Tournament Schedule
 
 | Event | Timestamp |
 | --: | :-- |
-| Registration Phase | 2018-10-26/2018-11-25 |
-| Live Drawings | 2018-11-26 (20:00) |
-| Qualifiers Stage | 2018-11-30/2018-12-04 |
+| Registration phase | 2018-10-26/2018-11-25 |
+| Live drawings | 2018-11-26 (20:00) |
```
Comment (TPGPL) [inline wiki/Tournaments/oSEAT/2/en.md]: add time zone

**Pull request merged**